bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/package-info.java

```diff
@@ -34,7 +34,7 @@
  * version, is bound to this proxy component resource type.
  * </p>
  */
-@Version("4.3.0")
+@Version("5.0.0")
```

Comment:
Why have you bumped major version ?

Reply:
CI suggested 5.0.0, should it be 4.4.0 then?